agent/util-scripts/pbench-tool-meister-start

```diff
@@ -244,7 +215,7 @@ def start_tms_via_ssh(
     redis_server,
```

Comment:
No type hint here? Isn't it a `RedisServerCommon`? Or maybe a `RedisServer`? Is it n ordering problem (the class is not defined yet)?

Reply:
Yes, ordering problem and I did not want to blow up the code with a major code move as part of the PR.

Reply:
Fixed in a separate commit.